From 55e09d562a1b8ca6a7d3ef2349c58f466bc44d2b Mon Sep 17 00:00:00 2001
From: Venture Research <tech@ventureresearch.com>
Date: Wed, 6 Feb 2013 20:51:02 -0600
Subject: [PATCH] hack to force use of libc malloc

Hack to force libc usage as it seems the option to pass it in has been
removed in favor of magic.

Note that this of course doesn't allow tcmalloc and jemalloc, however
jemalloc wasn't building correctly.

Upstream-Status: Pending

Signed-off-by: Venture Research <tech@ventureresearch.com>

Update to work with 4.0.8
Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

Reworked for 8.0.0
Signed-off-by: Eric Meyers <eric.meyers@arthrex.com>

---
 src/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 948ef83..5dbbefc 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -17,7 +17,8 @@
 # Just use 'make dep', but this is only needed by developers.
 
 release_hdr := $(shell sh -c './mkreleasehdr.sh')
-uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+# use fake uname option to force use of generic libc
+uname_S := "USE_LIBC_MALLOC"
 uname_M := $(shell sh -c 'uname -m 2>/dev/null || echo not')
 CLANG := $(findstring clang,$(shell sh -c '$(CC) --version | head -1'))
 
